### PR TITLE
Feature/fix empty json

### PR DESCRIPTION
**Pull request by suhaibMo**
In this PR,
- Lsf submissions to RH74
- flag -n to download ena mappings for fastq report for sequencing based studies
- single cell db connection
- Environmental variable for fastq report file ENA.

### CODE REVIEW

File: bin/geo_import_cron.sh [hunk 1]
```diff
@@ -99,7 +99,7 @@ pushd $supportingFilesPath
     filterGEOImport $dbConnection geo_${bulkORsinglecell}_rnaseq.tsv > latest_${bulkORsinglecell}_geo_accessions.txt
 
     ## Download GEO import soft files and convert to MAGE-TAB format (IDF and SDRF)
-    bsub -q production-rh7 -cwd `pwd` -M 80000 -R "rusage[mem=80000]" -o ${bulkORsinglecell}_geo_import_$today.out -e ${bulkORsinglecell}_geo_import_$today.err "$projectRoot/bin/import_geo_subs.pl -x -f latest_${bulkORsinglecell}_geo_accessions.txt -o $outputPath"
+    bsub -q production-rh74 -cwd `pwd` -M 80000 -R "rusage[mem=80000]" -o ${bulkORsinglecell}_geo_import_$today.out -e ${bulkORsinglecell}_geo_import_$today.err "$projectRoot/bin/import_geo_subs.pl -n -x -f latest_${bulkORsinglecell}_geo_accessions.txt -o $outputPath"
```
Comment (pcm32) [inline bin/geo_import_cron.sh]: although not part of the PR, do we really need to ask for 80 GB of RAM for downloading files?

File: bin/geo_import_magetab_eligibility.sh [hunk 2]
```diff
@@ -76,7 +76,16 @@ dbConnection=$(get_db_connection $pgAtlasUser $dbIdentifier)
 if [ $? -ne 0 ]; then
    "ERROR: dbConnection connection not established" >&2    
     exit 1
-fi 
+fi  
+
+# Set up sc DB connection details
+if  [ "$bulkORsinglecell" == 'singlecell' ]; then
+    scdbConnection=$(get_pg_db_connection -u 'atlasprd3' -d 'pro' -t 'scxa')
```
Comment (pcm32) [inline bin/geo_import_magetab_eligibility.sh:83]: question: is this using somewhere the environment files that we use for single cell environments? Otherwise you run the risk of pointing to wrong databases. Unless that these are not the same DBs of course that the web application and loading process for single cell use.

**Review by pcm32 — APPROVED** [review]
Looks good to me, just a few questions that might help improve this.

File: bin/geo_import_cron.sh (hunk 1 above)
Comment (suhaibMo) [reply bin/geo_import_cron.sh]: Relatively Yes, this sends lots runs consuming memory.

File: bin/geo_import_magetab_eligibility.sh (hunk 2 above)
Comment (suhaibMo) [reply bin/geo_import_magetab_eligibility.sh:83]: Yes, we store the prod db connection connection details in `$ATLAS_PROD/sw` so this function establishes connection for gxa and scxa dbs.

File: bin/geo_import_cron.sh (hunk 1 above)
Comment (suhaibMo) [reply bin/geo_import_cron.sh]: Actually doesn't consume so much we need to dynamically allocate to be efficient. I had used this to be on safe side as previously few runs failed.

**Comment by suhaibMo** [comment]
Thanks @pcm32 for the review and suggestions.
